### DISCUSSION TITLE
Can I upload the data of functions in old version applications?

### DISCUSSION THREAD

**Discussion by can-non** [naim94a/lumen#26] [Q&A] [answered]
Hi.
I'm currently reversing an old application which imported some old jpeg, png, ogg, and libvorbis libraries.
Those functions in the libraries are not just up-to-date, but even in very old version.(libvorbis 1.0.0)

I have some mind using this server as cloud but, if I upload older version of these libraries, those who pull the data from this server might get ridiculous result.
Can I upload these old functions to lumen server despite those data may not be suitable to upper version?

**Comment by naim94a** [comment, marked as answer]
I think it should be fine, wouldn’t the function hashes be different?
